### DISCUSSION TITLE
Feature request: Single user mode (avoid login screen)

### DISCUSSION THREAD

**Discussion by kcubeterm** [go-shiori/shiori#352] [Feature requests]
Is there any way to disable password in web interface. I just don't want password there.

**Comment by cauerego** [comment]
there's at least one way now

https://github.com/go-shiori/shiori/pull/290

**Comment by cauerego** [comment]
we kind of stop using shiori ever since... mostly because we couldn't possibly merge functionalities.

ideally we would convert all current crawling efforts into creating html pages for shiori! 😁

but we need agpl3+ and basically any other language, because we can't trust google with anything.

please, let us know if you can consider this or is willing to discuss such a monumental movement!

i do dearly miss shiori simplicity and ux!! 😏

**Reply by cauerego** [reply]
https://talk.ahoxus.org/notice/AGMIlOUZclMTcKZZia#agpl#gnu#jxself
